lib/server-builder.js

```diff
@@ -74,7 +74,7 @@ class ServerBuilder {
 
   getCommand () {
     const cmd = system.isWindows() ? 'gradlew.bat' : './gradlew';
-    const buildProperty = (key, value) => value ? `-P${key}=${value}` : null;
+    const buildProperty = (key, value) => value ? `-P${key}="${value}"` : null;
```

Comment:
is this only needed on Windows?
what is going to happen if the value already contains quotes?

Reply:
> is this only needed on Windows?

No, macOS also got the same error with `&` without `"` or `'`. The description's error was on macOS.

On Windows' case was (the error message was in Japanese tho) in powershell:
```
PS C:\Users\fly49\Downloads\appium-espresso-driver-master\appium-espresso-driver-master\espresso-server> ./gradlew.bat -PappiumKeyPassword=START&END
発生場所 行:1 文字:40
+ ./gradlew.bat -PappiumKeyPassword=START&END
+                                        ~
アンパサンド (&) 文字は許可されていません。& 演算子は、今後使用するために予約されています。アンパサンドを二重引用符で囲
み ("&")、文字列の一部として渡してください。
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : AmpersandNotAllowed
```

The above command worked in command prompt in Windows though.

`"xxxx\"yyyy"` or `"xxxx"yyyy"` like this string case the argument is divided with `"xxx"` and `"yyy"`. So in this case, we can use `'` intead. But probably no good way to avoid `"` and `'` at the same time.

Reply:
mm, i need more dig Win env... changed to draft.

Reply:
Hm, possibly we should set such value as ` ./gradlew.bat -PappiumKeyPassword\='START\"&\"END'` for Windows additionally. `"` itself needs in general.
Asking in #752

Reply:
you could probably use a check similar to https://github.com/appium/appium-adb/blob/86807a3dbde31c4c2ae49e0fddfeafad368d6bdc/lib/tools/adb-commands.js#L579

Reply:
It seems not all, but it seems safe to escape characters in ADB_INPUT_ESCAPE_PATTERN with `\"`. (at least, I got error messages by `<>|&`). So, chars in the pattern will be `'START\"&\"END'` (`\"`).